Update release notes for 8.1.1 (#13907)
elastic/logstash: docs/static/releasenotes.asciidoc

* <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
* <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>

[[logstash-8-1-1]]
=== Logstash 8.1.1 Release Notes

==== Notable issues fixed

> * Uninstalling a plugin with `bin/logstash-plugin uninstall <plugin>` will no longer fail https://github.com/elastic/logstash/pull/13823[#13823]

Reviewer: ```suggestion
* The`bin/logstash-plugin uninstall <plugin>` command works as expected, successfully uninstalling the specified plugin  https://github.com/elastic/logstash/pull/13823[#13823]
```

Reviewer: Our writing guidelines encourage writing to the positive

> * Uninstalling a plugin with `bin/logstash-plugin uninstall <plugin>` will no longer fail https://github.com/elastic/logstash/pull/13823[#13823]

Author: (Adding space to your suggestion so I can commit them all in one go from here)
```suggestion
The `bin/logstash-plugin uninstall <plugin>` command works as expected, successfully uninstalling the specified plugin https://github.com/elastic/logstash/pull/13823[#13823]
```

* Logstash CLI tools are now able to use the selected JDK on Windows https://github.com/elastic/logstash/pull/13839[#13839]

> * Logstash will no longer fail to find JVM located in a path including spaces on Windows https://github.com/elastic/logstash/pull/13881[#13881]

Reviewer: ```suggestion
* Logstash can successfully locate the Windows JVM, even if the path includes spaces https://github.com/elastic/logstash/pull/13881[#13881]
```

* The GeoIP database lookup will now respect a proxy defined with the http_proxy environment variable. https://github.com/elastic/logstash/pull/13840[#13840]

==== Updates to dependencies

* The version of the bundled JDK has been updated to 11.0.14.1+1. https://github.com/elastic/logstash/pull/13869[#13869]

==== Plugins

*Dissect Filter - 1.2.5*

* Fix: bad padding `->` suffix with delimiter https://github.com/logstash-plugins/logstash-filter-dissect/pull/84[#84]

*Elasticsearch Filter - 3.11.1*

* Fix: hosts => "es_host:port" regression https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/156[#156]

*Dead_letter_queue Input - 1.1.11*

* Fix: pre-flight checks before creating DLQ reader https://github.com/logstash-plugins/logstash-input-dead_letter_queue/pull/35[#35]
* Fix: avoid Logstash crash on shutdown if DLQ files weren't created https://github.com/logstash-plugins/logstash-input-dead_letter_queue/pull/33[#33]

*Elasticsearch Input - 4.12.2*

* Fix: hosts => "es_host:port" regression https://github.com/logstash-plugins/logstash-input-elasticsearch/pull/168[#168]

*Http_poller Input - 5.2.1*

> * Deps: unpin rufus-scheduler dependency https://github.com/logstash-plugins/logstash-input-http_poller/pull/130[#130]

Reviewer: Links to an unmerged draft PR. Not sure how or why this one got picked up. Changelog and PR numbers seem to check out, so 🤷

> * Deps: unpin rufus-scheduler dependency https://github.com/logstash-plugins/logstash-input-http_poller/pull/130[#130]

Author: ```suggestion
* Deps: unpin rufus-scheduler dependency https://github.com/logstash-plugins/logstash-input-http_poller/pull/132[#132]
```
